code/game/objects/items/holosign_creator.dm

```diff
@@ -96,45 +117,49 @@
 	desc = "A holographic projector that creates PENLITE holobarriers. Useful during quarantines since they halt those with malicious diseases."
 	icon_state = "signmaker_med"
 	holosign_type = /obj/structure/holosign/barrier/medical
+	sign_name = "holobarrier"
 	creation_time = 30
 	max_signs = 3
 
 /obj/item/holosign_creator/cyborg
-	name = "Energy Barrier Projector"
+	name = "\improper Energy Barrier Projector"
```

Comment:
if you could just remove the improper and uncapitalize this

Reply:
done